finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,150 @@
+Fetch account transactions
+===
+Using the Finance API, you can fetch all transactions or transactions of certain types from a merchant's Zettle liquid account during a specific period.
+
+> **Note**: A merchant's Zettle liquid account contains all confirmed transactions that are already paid out or to be paid out to the merchant's bank account. If you want to check preliminary transactions that are being confirmed with buyers' banks, you can fetch them from a merchant's Zettle preliminary account.
+
+<!-- Is there any limit for how old transactions can be fetched? -->  
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Fetch all transactions during a specific period](#step-1-fetch-all-transactions-during-a-specific-period)
+* [Step 2: Fetch transactions of certain types during a specific period](#step-2-fetch-transactions-of-certain-types-during-a-specific-period)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Step 1: Fetch all transactions during a specific period
```

Comment:
To me it's contradicting. When i fetch transactions for a specific period means that i don't fetch all of them only a selection. 
Suggestion:
 _Fetch all types of transactions during a specific period_
 
 Or 
 
 _Fetch transactions during a specific period_